sig-list.md

```diff
@@ -40,8 +40,8 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
 |[Network](sig-network/README.md)|* [Tim Hockin](https://github.com/thockin), Google<br>* [Dan Williams](https://github.com/dcbw), Red Hat<br>* [Casey Davenport](https://github.com/caseydavenport), Tigera<br>|* [Slack](https://kubernetes.slack.com/messages/sig-network)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-network)|* [Thursdays at 21:00 UTC (biweekly)](https://zoom.us/j/5806599998)<br>
 |[Node](sig-node/README.md)|* [Dawn Chen](https://github.com/dchen1107), Google<br>* [Derek Carr](https://github.com/derekwaynecarr), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-node)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-node)|* [Tuesdays at 17:00 UTC (weekly)](https://plus.google.com/hangouts/_/google.com/sig-node-meetup?authuser=0)<br>
 |[On Premise](sig-on-premise/README.md)|* [Marco Ceppi](https://github.com/marcoceppi), Canonical<br>* [Dalton Hubble](https://github.com/dghubble), CoreOS<br>|* [Slack](https://kubernetes.slack.com/messages/sig-onprem)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-on-prem)|* [Wednesdays at 16:00 UTC (weekly)](https://zoom.us/my/k8s.sig.onprem)<br>
-|[OpenStack](sig-openstack/README.md)|* [Ihor Dvoretskyi](https://github.com/idvoretskyi), CNCF<br>* [Steve Gordon](https://github.com/xsgordon), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-openstack)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-openstack)|* [Wednesdays at 15:00 UTC (biweekly)](https://zoom.us/j/417251241)<br>
-|[Product Management](sig-product-management/README.md)|* [Aparna Sinha](https://github.com/apsinha), Google<br>* [Ihor Dvoretskyi](https://github.com/idvoretskyi), CNCF<br>* [Caleb Miles](https://github.com/calebamiles), CoreOS<br>|* [Slack](https://kubernetes.slack.com/messages/kubernetes-pm)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-pm)|* [Tuesdays at 15:00 UTC (biweekly)](https://zoom.us/j/845373595)<br>
+|[OpenStack](sig-openstack/README.md)|* [Ihor Dvoretskyi](https://github.com/idvoretskyi), Mirantis<br>* [Steve Gordon](https://github.com/xsgordon), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-openstack)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-openstack)|* [Wednesdays at 15:00 UTC (biweekly)](https://zoom.us/j/417251241)<br>
```

Comment:
Found after doing `make`.

Reply:
Sorry.. this was caused by another PR. You should be able to rebase, and it will go away

Reply:
Gone as you said @cblecker . Thanks.